Parser/tokenizer.c

```diff
@@ -2272,7 +2274,7 @@ tok_get_fstring_mode(struct tok_state *tok, tokenizer_mode* current_tok, const c
     }
 
     *p_start = tok->start;
-    *p_end = tok->cur-current_tok->f_string_quote_size;
+    *p_end = tok->cur - 1;
```

Comment:
Ideally, we should be doing `*p_end = tok->cur` here, which breaks parsing f-strings though. Any thoughts on why this is and why this was done like this in the first place?

Reply:
Yup, I understand that. My question is why isn't it `*p_end = tok->cur`. IIRC `p_end` points to the character just after the current token. For example, when parsing `'abcde'\n`, the tokenizer will emit a `STRING` node and `p_end` will point at the newline character just after it. Why do we want to point the quote character (the first quote character for triple-quoted strings) instead? Was there a reasoning behind this?